qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -333,6 +333,7 @@
             no RHEL.6, RHEL.7
             gagent_check_type = os_basic_info
             cmd_get_host_name = hostname
+            cmd_set_host_name = hostnamectl set-hostname newhostname && cat /etc/hostname
```

Comment:
Is this works on Windows guest?

Reply:
thanks for reviewing.
this patch was fixed linux bug and we covered it into polarion ,but there are no steps about windows_guest situation,so I didn't consider windows guest cmd here.

Reply:
I‘ll add 'if' condition to filter linux os.what do you think?

Reply:
@6-dehan Yes please, otherwise it might fail on Windows guest.

Reply:
@vivianQizhu 
If condition has been updated,check please.